### PR TITLE
Updated FunctionTable and implement some function

### PR DESCRIPTION
**Pull request by JamePeng**
Updated some module's FunctionTable from 3dbrew
Correct AppletUtility() and LoadComponent() parameter!
Align some code
Implement ReplySleepQuery() and ListDataTitleTicketInfos() 
Implement function GetOutputFormat() and GetInputFormat()

### CODE REVIEW

**Comment by neobrain** [comment]
Hi, this is neobot, using neobrain's account. I'm keeping an archive of versions of this PR:

2016-02-09T09:57:09Z: https://github.com/citra-emu/citra/compare/f1d1049c4fdf51fd382f9afb6b5cd5f104e71b0e...JamePeng:5ed3824d40639a06caf148e478bee36e33025e88

**Comment by bunnei** [comment]
Please change the commit message to be descriptive of the changes and grammatically correct. Also, it's "3dbrew" not "3dbrez".

File: src/core/hle/service/apt/apt.h [hunk 1]
```diff
@@ -46,24 +46,34 @@ enum class SignalType : u32 {
 enum class AppletId : u32 {
     HomeMenu           = 0x101,
     AlternateMenu      = 0x103,
-    Camera             = 0x110,
-    FriendsList        = 0x112,
-    GameNotes          = 0x113,
+    CameraApplet       = 0x110,
```
Comment (bunnei) [inline src/core/hle/service/apt/apt.h]: What's the reason for appending 'Applet' to all of these?

File: src/core/hle/service/apt/apt_a.cpp [hunk 2]
```diff
@@ -9,25 +9,95 @@ namespace Service {
...
+    {0x00070000, nullptr,                     "GetLastSignaledAppletId" },
+    {0x00080000, nullptr,                     "CountRegisteredApplet" },
+    {0x00090040, IsRegistered,                "IsRegistered" },
+    {0x000A0040, nullptr,                     "GetAttribute" },
+    {0x000B0040, InquireNotification,         "InquireNotification" },
+    {0x000C0104, SendParameter,               "SendParameter" },
+    {0x000D0080, ReceiveParameter,            "ReceiveParameter" },
+    {0x000E0080, GlanceParameter,             "GlanceParameter" },
+    {0x000F0100, CancelParameter,             "CancelParameter" },
+    {0x001000C2, nullptr,                       "DebugFunc" },
```
Comment (bunnei) [inline src/core/hle/service/apt/apt_a.cpp]: The columns should all be aligned

File: src/core/hle/service/apt/apt.h (hunk 1 above)
Comment (JamePeng) [inline src/core/hle/service/apt/apt.h]: I think it's a much better description of ID "What is this?"

Comment (purpasmart96) [inline src/core/hle/service/apt/apt.h]: The name of the enum makes kinda redundant

**Comment by neobrain** [comment]
Hi, this is neobot, using neobrain's account. I'm keeping an archive of versions of this PR:

2016-02-10T07:01:42Z: https://github.com/citra-emu/citra/compare/f1d1049c4fdf51fd382f9afb6b5cd5f104e71b0e...JamePeng:539b072abfb2d68e70052be0a8819816d599993e

**Comment by neobrain** [comment]
Hi, this is neobot, using neobrain's account. I'm keeping an archive of versions of this PR:

2016-02-10T18:10:12Z: https://github.com/citra-emu/citra/compare/f1d1049c4fdf51fd382f9afb6b5cd5f104e71b0e...JamePeng:3b0a54437c0d473202c88cf0d1dcfc41922e3ad7

**Comment by lioncash** [comment]
This PR is doing more than just updating the function table.

**Comment by JamePeng** [comment]
@lioncash That is not accurate,
With my operation of updatiing function table!
I found some function could be implement , some "guess function" should be corrected

**Comment by lioncash** [comment]
@JamePeng Please separate these changes into two different PR's. 

A PR should do one thing as its main overall task. One should be the updating of the table itself. The other one should be about the implementations of the functions along with providing any necessary documentation.

**Comment by JamePeng** [comment]
@lioncash How about I repull request again?

**Comment by lioncash** [comment]
@JamePeng You can just use `git cherry-pick` to move the function implementation commits into another branch, then use `git reset` to roll the branch this PR is on back to just the table updating commit.

**Comment by neobrain** [comment]
@jamepeng: Please stop creating new PRs just because you mess up something about the other ones. It's just pointless spam, especially for those of us who are subscribed to every repository activity.

**Pull request closed without merging**